code/src/App.js

```diff
@@ -1,9 +1,22 @@
 import React from 'react'
```

Comment:
Just tested out your project a bit - I really like it, but one thing I was missing was localstorage! It was actually easier to implement than I thought it would be.

https://www.npmjs.com/package/redux-localstorage
npm i redux-localstorage

It only requires a couple changes to the code in App.js - I'll show you how to do it below: